spec/support/acts_as_tenant.rb

```diff
@@ -9,6 +9,7 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        Capybara.app_host = "http://" + send(tenant).host
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.